src/main/resources/xlr_xldeploy/getAllVersionsTask.py

```diff
@@ -22,5 +22,12 @@
 
 packageIds = xld_client.get_all_package_version(applicationId)
 
+if stripApplications:
+    packageIdsTmp = list()
+    for version in packageIds:
```

Comment:
I think you can do this in a one liner:
`packageIds = [version.partition('/')[2] for version in packageIds]`

Also, will this work when the application is under a `udm.Directory`?

Reply:
Thanks, I just tested your one liner modification and it is working.
I also just tested with an application under a `core.Directory` and it is working well.